Core/tlds.js

```diff
@@ -0,0 +1,7921 @@
+//
+//  tlds.js
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+// Based on https://github.com/131/node-tld
+var duckduckgoTLDParser = function() {
+
+	var tlds = {
```

Comment:
I'm a bit worried about updating this for two reasons:
• it's been modified so we can't just drop in a new version to update it
• we have another copy of the list (in the official format) in the app. It would be easy to update one file and forget the other

Can we use the tld file already in the codebase and parse it into the format you need for the js layer?

Reply:
Happy for to be capture this in a separate Asana task if this isn't a good time to tackle it.

Reply:
We should also be downloading a new version of it from somewhere IMHO. Suggest we defer this to later, it's not going to update often.

Reply:
Cool, I see your new Asana task capturing this.